Universal FE Randomizer/src/util/OptionRecorder.java

```diff
@@ -282,6 +284,7 @@ public static void recordGBAFEOptions(FEBase.GameType gameType, GrowthOptions gr
 		bundle.itemAssignmentOptions = itemAssignment;
 		bundle.seed = seed;
 		bundle.version = GBAOptionBundleVersion;
+		bundle.characterShufflingOptions = shufflingOptions;
```

Comment:
If you update the bundle, you should update the version too at the top of this file so it can correctly save and load options from previous runs.